en/docs/tutorials/scenarios/scenario1-create-rest-api.md

```diff
@@ -1,4 +1,75 @@
 # Scenario 1 - Create REST API from an OpenAPI Definition
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**Coltrain is one of the railway companies that is partnered up with GOGO Train to provide better service to their customers. Coltrain already has some internally managed APIs deployed in-house and these are managed by their internal development team. One of the APIs is train schedule retrieval API which is intended for the public community to get the Coltrain schedules. Currently, this API is exposed for the public and Coltrain development team faces challenges in maintaining and handling high load for the API . By exposing this API through WSO2 API Manager, Coltrain expects to get the full benefits of an API Management solution such as API lifecycle management, security, throttling, etc. and decouple the maintenance overhead from the internal teams.**
```

Comment:
Let's not bold this. It has a negative impact on readability.

Reply:
fixed